315.md

```diff
@@ -0,0 +1,60 @@
+
+NIP-315
+=======
+
+User Statuses
+--------------
+
+`draft` `optional` `author:jb55`
+
+## Abstract
+
+This NIP enables a way for users to share live statuses such as what music they are listening to, as well as what they are currently doing: work, play, out of office, etc.
+
+## Live Statuses
+
+A special event with `kind:30315` "User Status" is defined as an *optionally expiring* _parameterized replaceable event_, where the `d` tag represents the status type:
+
+For example:
+
+```js
+{
+  "kind": 30315,
+  "content": "Sign up for nostrasia!",
+  "tags": [
+    ["d", "general"],
+    ["r", "https://nostr.world"]
+  ],
+}
+
+{
+  "kind": 30315,
+  "content": "Intergalatic - Beastie Boys",
```

Comment:
Does this allow for empty text, text containing line breaks, or long text?

Reply:
Its allowed but would be cut off in Damus

Reply:
Empty statuses are equivalent to clearing the status in Damus, could make that explicit